### PR TITLE
Created landing page for "Microsoft Edge Developer documentation"

### PR DESCRIPTION
**Pull request by mikehoffms**
- The Content & Learning/ Information Architecture team has signed-off on this design, including the landing page, the breadcrumb path, and the top of the TOC.
- There is one link to the hub page: it's in the breadcrumb path, as 3 words: "Microsoft Edge documentation".
- Experimentation showed that the tools are designed to support this standard design.  The top node of the TOC is special and reserved for a landing page and the nodes below are kept from moving to the right and costing several characters.
- Tested links on hub and landing.
- Out of scope: Redesign hub page cards for balanced TOC coverage.  [Microsoft Edge documentation](https://docs.microsoft.com/en-us/microsoft-edge/)  That's potentially trivial: copy the landing page cards to the hub page article and search/replace to correct the syntax.

Rendered page for review: 
[Microsoft Edge Developer documentation](https://review.docs.microsoft.com/en-us/microsoft-edge/developer/index?branch=pr-en-us-1419) landing page

Compare:
[Microsoft Edge Enterprise documentation](https://docs.microsoft.com/en-us/deployedge/) landing page
The Content & Learning/ Information Architecture team plans to change the Enterprise landing page, breadcrumb path, and top-of-TOC design in line with this Developer docs design.

[Existing public docs](https://docs.microsoft.com/en-us/microsoft-edge/devtools-guide-chromium/) - no landing page

### CODE REVIEW

File: microsoft-edge/developer/index.yml [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+            url: /microsoft-edge/devtools-guide-chromium
+          - text: DevTools for beginners
+            url: /microsoft-edge/devtools-guide-chromium/beginners/html
+      - linkListType: how-to-guide
+        links:
+          - text: Command menu
+            url: /microsoft-edge/devtools-guide-chromium/command-menu/index
+          - text: Open DevTools
+            url: /microsoft-edge/devtools-guide-chromium/open
+          - text: CSS
```
Comment (captainbrosset) [inline microsoft-edge/developer/index.yml:51]: Can we maybe find a more self-explanatory title for this section? How about "View and change CSS"?

**Review by captainbrosset — APPROVED** [review]
Great landing page! I just left one comment in the code.
Additionally, I found a few broken links, I wonder why they weren't reported automatically:
- Accessibility testing
- Accessibility testing features in DevTools
- DevTools extension for Visual Studio Code
- webhint extension for Visual Studio Code
- Detecting Microsoft Edge from your website
- Performance analysis
There might be others, but I think I checked most.
I think this comes from the fact that `.md` is in the url, and it should not.

**Comment by mikehoffms** [comment]
@captainbrosset 
> Additionally, I found a few broken links, I wonder why they weren't reported automatically:
>   I think this comes from the fact that `.md` is in the url, and it should not.

omg you just indirectly found a broken link in the public hub page
https://docs.microsoft.com/en-us/microsoft-edge/#microsoft-edge-for-developers - click "Add and manage users".
Found b/c it's the only ".md" in the hub page source file.

File: microsoft-edge/index.yml [hunk 2]
```diff
@@ -193,7 +193,7 @@ additionalContent:
             - text: Open developer account
               url: /microsoft-edge/extensions-chromium/publish/create-dev-account
```
Comment (mikehoffms) [inline microsoft-edge/index.yml:194]: performance analysis link is broken on landing and hub page though .md isn't included

File: microsoft-edge/developer/index.yml [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+          - text: Microsoft Edge WebView2 API Reference
+            url: /microsoft-edge/webview2/webview2-api-reference
+
+  # Card
+  - title: Test & Automation
+    linkLists:
+      - linkListType: how-to-guide
+        links:
+          - text: Accessibility
+            url: /microsoft-edge/accessibility
```
Comment (Reezaali) [inline microsoft-edge/developer/index.yml:126]:
```suggestion

```

File: microsoft-edge/developer/index.yml [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+        links:
+          - text: Accessibility
+            url: /microsoft-edge/accessibility 
+          - text: DevTools Protocol overview
+            url: /microsoft-edge/devtools-protocol-chromium
+          - text: WebDriver
+            url: /microsoft-edge/webdriver-chromium
+          - text: Puppeteer
+            url: /microsoft-edge/puppeteer
+          - text: Site compatibility-impacting changes
```
Comment (Reezaali) [inline microsoft-edge/developer/index.yml:133]: Can we move the site compat link to the Web Platform card? Thanks.

**Review by Reezaali — CHANGES_REQUESTED** [review]

**Review by Reezaali — APPROVED** [review]

**Pull request merged**